tests/acceptance/view-user-profile-test.js

```diff
@@ -63,6 +81,11 @@ module('Acceptance | view-user-profile', function (hooks) {
     });
 
     await userPage.visit({ username: 'rohitpaulk' });
+    assert.strictEqual(userPage.courseLabels[0].text, 'Build your own Docker');
+    // Completed course will be listed first.
```

Comment:
Keep comments above the lines that they apply to - isn't Docker the completed one here?